tox.ini

```diff
@@ -251,7 +251,7 @@ commands=
 
 [ruff]
 deps=
-    ruff
+    ruff==0.0.258
```

Comment:
The script will only update dependencies pinned in `tox.ini`, so we need to pin ruff here if we want it to be updated regularly.